content/en/community/_index.html

```diff
@@ -1,69 +1,243 @@
----
-title: Community
-layout: basic
-cid: community
----
+---
+title: Community
+layout: basic
+cid: community
+---
+
+<div class="newcommunitywrapper">
+<div class="banner1">
+  <img src="/images/community/kubernetes-community-final-02.jpg" style="width:100%" class="desktop">
+  <img src="/images/community/kubernetes-community-02-mobile.jpg" style="width:100%" class="mobile">
+</div>
+
+<div class="intro">
+
+<p>The Kubernetes community -- users, contributors, and the culture we've built together -- is one of the biggest reasons for the meteoric rise of this open source project. Our culture and values continue to grow and change as the project itself grows and changes. We all work together toward constant improvement of the project and the ways we work on it.
+<br><br>We are the people who file issues and pull requests, attend SIG meetings, Kubernetes meetups, and KubeCon, advocate for it's adoption and innovation, run <code>kubectrl get pods</code>, and contribute in a thousand other vital ways. Read on to learn how you can get involved and become part of this amazing&nbsp;community.</p>

```

Comment:
This should be `kubectl get pods`, right?
And I think we can remove `&nbsp;` on the last line.